DOC-1256 clarify status of zoning and offloading (#1285)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/cross-server-rpcs.md

In native-Unreal networking, [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs) are functions which either the client or the server use to send messages to each other over a network connection. 

> Cross-server RPCs are a custom solution to take advantage of the [zoning]({{urlRoot}}/content/glossary#zoning), which is one of the GDK's options for multiserver development.

Reviewer: Good spot!!!!

```suggestion
Cross-server RPCs facilitate [zoning]({{urlRoot}}/content/glossary#zoning), which is one of the GDK's options for multiserver development.
```
Think this is probs better like this?

Author: Aah, good point - yeah, they facilitate zoning rather than taking advantage of it

> **Note:** Support for zoning is currently in pre-alpha. We invite you to try out the [multiserver zoning shooter tutorial]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro) and learn about how it works, but we don’t recommend you start developing features that use zoning yet.

> In Unreal’s native single-client architecture, your game server holds the canonical state of the whole game world. As there is a single game server, it has complete authority over all server Actors and so it is able to invoke and execute functions on Actors unhindered. 

Reviewer: ```suggestion
In Unreal’s native single-server architecture, your game server holds the canonical state of the whole game world. As there is a single game server, it has complete authority over all server Actors and so it is able to invoke and execute functions on Actors unhindered. 
```

Reviewer: Single client wouldn't make very interesting multiplayer games.  <chortle>

Author: 🤦‍♂
